### PR TITLE
fix(learn) : update the quiz cert footer

### LINKED ISSUE #54282: Update certification footer for exam-based certifications
On exam-based certifications, we should change the footer verbiage slightly to reflect the fact that it's an exam (and not submitted projects). See screenshot below.

freecodecamp_org_certification_harrykimpel_foundational-c-sharp-with-microsoft

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #54282

The wording for the description of the certification has been updated and the footer is now conditionally not present for the C# exam certification.

### CODE REVIEW

File: client/src/client-only-routes/show-project-links.tsx [hunk 1]
```diff
@@ -158,6 +158,16 @@ const ShowProjectLinks = (props: ShowProjectLinksProps): JSX.Element => {
   const { completedChallenge, showCode, projectTitle } = solutionState;
   const examResults = completedChallenge?.examResults;
 
+  const getCertHanding = (certName: string) => {
```
Comment (naomi-lgbt) [inline client/src/client-only-routes/show-project-links.tsx:161]:
```suggestion
  const getCertHeading = (certName: string) => {
```

File: client/src/client-only-routes/show-project-links.tsx [hunk 2]
```diff
@@ -176,12 +186,7 @@ const ShowProjectLinks = (props: ShowProjectLinksProps): JSX.Element => {
 
   return (
     <div data-cy='solution-widget' data-playwright-test-label='project-links'>
-      {t(
-        certName === 'Legacy Full Stack'
-          ? 'certification.project.heading-legacy-full-stack'
-          : 'certification.project.heading',
-        { user: name }
-      )}
+      {t(getCertHanding(certName), { user: name })}
```
Comment (naomi-lgbt) [inline client/src/client-only-routes/show-project-links.tsx:189]:
```suggestion
      {t(getCertHeading(certName), { user: name })}
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Not sure what a handing is, but probably not something either of us want.

**Comment by a2937** [comment]
Typo has been fixed.

**Comment by Sembauke** [comment]
Hey @a2937,

Thank you for making this pr. Some of the test are failing could you take a look?

If you need any help with resolving the test errors let me know!

**Comment by a2937** [comment]
image

I don't understand why the tests are failing on GitHub. While I did modify my local development user account to gain this certificate (plus did a quick change back and forth on the tests) nothing seemed broken. The only reason why the tests failed locally were because the issuing dates were different the seed data results.

File: e2e/certification.spec.ts [hunk 3]
```diff
@@ -213,11 +213,11 @@ test.describe('Certification page - Microsoft', () => {
       '/learn/foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam'
     );
 
-    await expect(
+    expect(
```
Comment (naomi-lgbt) [inline e2e/certification.spec.ts:216]:
```suggestion
    await expect(
```

File: e2e/certification.spec.ts [hunk 4]
```diff
@@ -213,11 +213,11 @@ test.describe('Certification page - Microsoft', () => {
...
       page.getByText(
         'If you suspect that any of these projects violate the academic honesty policy, please report this to our team.'
       )
-    ).toBeVisible();
+    ).toBeNull();
```
Comment (naomi-lgbt) [inline e2e/certification.spec.ts:220]:
```suggestion
    ).toHaveCount(0);
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
I had weirdness with `toBeNull()`.

**Review by naomi-lgbt — APPROVED** [review]

**Review by jdwilkin4 — APPROVED** [review]

**Pull request merged**